### PR TITLE
correct input aliasing

### PR DESCRIPTION
**Pull request by ryanfkeepers**
Updates and corrects input aliasing according to
the following rules (in priority order):
1. if the library name is usable, use it
2. if not, alias to the package name
3. if the package name is weird, alias sensibly
4. in case of collision, alias more distant imports
5. aliases should be consistent throughout

---

#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [x] :broom: Tech Debt/Cleanup

#### Issue(s)

* #1970

#### Test Plan

- [x] :zap: Unit test

### CODE REVIEW

File: src/internal/connector/mockconnector/mock_data_message.go [hunk 1]
```diff
@@ -5,13 +5,12 @@ import (
 	"fmt"
 	"testing"
 
-	absser "github.com/microsoft/kiota-abstractions-go/serialization"
-	js "github.com/microsoft/kiota-serialization-json-go"
+	"github.com/alcionai/clues"
+	serialization "github.com/microsoft/kiota-abstractions-go/serialization"
```
Comment (meain) [inline src/internal/connector/mockconnector/mock_data_message.go:9]: Suggestion:

```suggestion
	kserialization "github.com/microsoft/kiota-abstractions-go/serialization"
```

File: src/internal/connector/mockconnector/mock_data_message.go [hunk 2]
```diff
@@ -5,13 +5,12 @@ import (
...
+	kioser "github.com/microsoft/kiota-serialization-json-go"
```
Comment (meain) [inline src/internal/connector/mockconnector/mock_data_message.go:10]: Suggestion:

```suggestion
	kjson "github.com/microsoft/kiota-serialization-json-go"
```

File: src/internal/connector/sharepoint/collection.go [hunk 3]
```diff
@@ -7,13 +7,13 @@ import (
 	"time"
 
 	"github.com/alcionai/clues"
-	absser "github.com/microsoft/kiota-abstractions-go/serialization"
-	kw "github.com/microsoft/kiota-serialization-json-go"
+	"github.com/microsoft/kiota-abstractions-go/serialization"
+	kioser "github.com/microsoft/kiota-serialization-json-go"
```
Comment (meain) [inline src/internal/connector/sharepoint/collection.go:11]: Same as previously mentioned if we are updating it.

**Review by meain — APPROVED** [review]
Just two suggestions, approving.

File: src/internal/connector/mockconnector/mock_data_message.go (hunk 1 above)
Comment (ryanfkeepers) [reply src/internal/connector/mockconnector/mock_data_message.go:9]: Oh, that shouldn't be aliased in the first place.

File: src/internal/connector/mockconnector/mock_data_message.go (hunk 2 above)
Comment (ryanfkeepers) [reply src/internal/connector/mockconnector/mock_data_message.go:10]: I dig it

**Pull request merged**